cpp/ql/test/library-tests/dataflow/fields/dataflow-consistency.expected

```diff
@@ -2,15 +2,7 @@ uniqueEnclosingCallable
 | C.cpp:37:24:37:33 | 0 | Node should have one enclosing callable but has 0. |
 | C.cpp:37:24:37:33 | new | Node should have one enclosing callable but has 0. |
 uniqueTypeBound
-| complex.cpp:22:11:22:17 | constructor init of field f [post-this] | Node should have one type bound but has 0. |
```

Comment:
Not that I'm complaining, but how did these get fixed by adding a new unused predicate? Does the extractor now emit the existing info for more functions as well?

Reply:
The extractor change means that `Bar *` is now in the database, and it wasn't before. Does that explain it?

Reply:
That must be because of this code, which expects to find a `C *` type in the db for every class `C` with a `ConstructorFieldInit`:

https://github.com/github/codeql/blob/beb66299e9c86b1312356743c79baef2181a216d/cpp/ql/src/semmle/code/cpp/dataflow/internal/DataFlowUtil.qll#L376-L378

Reply:
OK, I believe that explanation.